File: Domain/Domain/Cliente.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Domain
{
    public class Cliente
    {
        public int Id { get; set; }
        [Required]
        [StringLength(100)]
        public string? Nombre { get; set; }
        [Required]
        [StringLength(100)]
        public string? Apellido { get; set; }
        [Required]
        public int Edad { get; set; }
        [Required]
        [StringLength(100)]
        public string? CorreoElectronico { get; set; }

        public List<Factura> Facturas { get; set; }
    }
}

File: ProyectoAngular/src/app/models/Cliente.ts
import { Factura } from "./Factura";

export interface Cliente
{
    id:number;
    nombre: string;
    apellido: string; 
    edad: number;
    correoElectronico: string; 
    Facturas:Factura[];
}

export interface IDelete{
    id:Number;
    user:string;
}